apps/core/scenes/search/PlacePicker/PlacePickerContent.js

```diff
@@ -110,11 +113,14 @@ class PlacePickerContent extends React.Component<Props, State> {
   };
 
   refetchSuggestions = debounce((text: string) => {
-    this.props.relay.refetch({ input: { term: text } });
+    this.props.relay.refetch({ input: { term: text } }, null, () => {
+      this.setState({ isLoading: false });
+    });
   }, DEBOUNCE_TIME);
 
   handleChangeText = (text: string) => {
     this.setState({ searchText: text });
+    this.setState({ isLoading: true });
```

Comment:
Can be merged with state update line above. Otherwise looks good.

Reply:
👍

Reply:
done